content/manuals/desktop/features/networking/networking-how-tos.md

```diff
@@ -12,7 +12,7 @@ aliases:
 - /docker-for-windows/networking/
 - /desktop/windows/networking/
 - /desktop/networking/
-weight: 30
+- /desktop/features/networking/
```

Comment:
I think this will cause an infinite redirect loop because new `_index.md` page you're adding will be served on this path.